assets/src/edit-story/components/canvas/navLayer.js

```diff
@@ -54,8 +52,7 @@ function NavLayer() {
     ev.stopPropagation();
   }, []);
 
-  const showQuickActions =
-    enableQuickActionMenu && !hasHorizontalOverflow && quickActions.length;
+  const showQuickActions = !hasHorizontalOverflow && quickActions.length;
```

Comment:
@samwhale Since you defaulted the flag to true le'ts leave all of this so that if we need to swap the flag to false it's still conditional.